pkg/meta/load_dump_test.go

```diff
@@ -113,6 +117,10 @@ func TestLoadDump(t *testing.T) {
 		if err = m.DumpMeta(fp); err != nil {
 			t.Fatalf("dump meta: %s", err)
 		}
+		cmd := exec.Command("diff", "redis.dump", "sqlite3.dump")
```

Comment:
compare sqlite3.dump with sampleFile directly might be better since it's a standalone test now